doc/codi.txt

```diff
@@ -264,6 +265,15 @@ g:codi#sync
 
              Default value is 0.
 
+                                                       *g:codi#use_buffer_dir*
+g:codi#use_buffer_dir
+             Whether or not switch to switch to the directory of the buffer
```

Comment:
just some minor wording cleanup here. `"Whether or not switch to switch..."` should/could/ought to read: `"Determines whether it should switch to the directory of the currently edited buffer before starting the REPL."`